vizier/backend/src/info/vizierdb/commands/plot/ScatterPlot.scala

```diff
@@ -65,9 +65,9 @@ object ScatterPlot extends Command
       DatasetParameter(id = PARAM_DATASET, name = "Dataset"),
       ColIdParameter(id = PARAM_X, name = "X-axis"),
       ColIdParameter(id = PARAM_Y, name = "Y-axis"),
+      NumericalFilterParameter(id = PARAM_FILTER, name = "Filter",required = false),
       StringParameter(id = PARAM_LABEL, name = "Label", required = false),
-      StringParameter(id = PARAM_FILTER, name = "Filter", required = false, helpText = Some("e.g., state = 'NY'")),
-      StringParameter(id = PARAM_COLOR, name = "Color", required = false, helpText = Some("e.g., #214478")),
+      ColorParameter(id = PARAM_COLOR, name = "Color", required = false),
```

Comment:
See notes above.  Also, Y seems to be a single value here... why isn't it consistent with the rest?